### PR TITLE
fix: Remove default imports from components

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Fixes: https://github.com/Workday/canvas-kit/issues/1315

category

### Release Note
We've updated some of our components to remove default exports. This matches what most of our component are doing.

### BREAKING CHANGES
Some component will now have to be imported like this: `import { export1 } from "module-name";`

---

## Checklist

- [ ] tests are changed or added
- [ ] code has been documented
- [ ] design approved final implementation
- [ ] a11y approved final implementation
- [ ] code adheres to the [API & Pattern guidelines](https://workday.github.io/canvas-kit/?path=/story/welcome-dev-docs-api-pattern-guidelines--page)

### CODE REVIEW

File: modules/react/radio/index.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import Radio from './lib/Radio';
+import {Radio} from './lib/Radio';
 
-export default Radio;
 export {Radio};
```
Comment (RayRedGoose) [inline modules/react/radio/index.ts:3]:
```suggestion
```

File: modules/react/radio/index.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
-export {default as RadioGroup} from './lib/RadioGroup';
+export {RadioGroup} from './lib/RadioGroup';
```
Comment (RayRedGoose) [inline modules/react/radio/index.ts:4]: Should it be removed as it's exported on line 6?

File: modules/react/select/index.ts [hunk 3]
```diff
@@ -1,7 +1,6 @@
-import Select from './lib/Select';
+import {Select} from './lib/Select';
```
Comment (RayRedGoose) [inline modules/react/select/index.ts:1]:
```suggestion
```

File: modules/react/select/index.ts [hunk 4]
```diff
@@ -1,7 +1,6 @@
...
 
-export default Select;
 export {Select};
```
Comment (RayRedGoose) [inline modules/react/select/index.ts:3]:
```suggestion
```

File: modules/react/select/index.ts [hunk 5]
```diff
@@ -1,7 +1,6 @@
...
-export {default as SelectOption} from './lib/SelectOption';
+export {SelectOption} from './lib/SelectOption';
```
Comment (RayRedGoose) [inline modules/react/select/index.ts:4]: Same here. It had sense when it was default export but not now

File: modules/react/switch/index.ts [hunk 6]
```diff
@@ -1,5 +1,4 @@
-import Switch from './lib/Switch';
+import {Switch} from './lib/Switch';
```
Comment (RayRedGoose) [inline modules/react/switch/index.ts:2]:
```suggestion
```

File: modules/react/switch/index.ts [hunk 7]
```diff
@@ -1,5 +1,4 @@
...
 
-export default Switch;
 export {Switch};
```
Comment (RayRedGoose) [inline modules/react/switch/index.ts:3]:
```suggestion
```

File: modules/react/text-input/index.ts [hunk 8]
```diff
@@ -1,7 +1,6 @@
-import InputIconContainer from './lib/InputIconContainer';
-import TextInput from './lib/TextInput';
+import {InputIconContainer} from './lib/InputIconContainer';
+import {TextInput} from './lib/TextInput';
```
Comment (RayRedGoose) [inline modules/react/text-input/index.ts:2]:
```suggestion
```

File: modules/preview-react/select/index.ts [hunk 9]
```diff
@@ -1,6 +1,5 @@
-import Select from './lib/Select';
+import {Select} from './lib/Select';
```
Comment (RayRedGoose) [inline modules/preview-react/select/index.ts:1]:
```suggestion
```

File: modules/preview-react/select/index.ts [hunk 10]
```diff
@@ -1,6 +1,5 @@
...
 
-export default Select;
 export {Select};
```
Comment (RayRedGoose) [inline modules/preview-react/select/index.ts:3]:
```suggestion
```

File: modules/labs-react/drawer/index.ts [hunk 11]
```diff
@@ -1,5 +1,4 @@
-import Drawer from './lib/Drawer';
-import DrawerHeader from './lib/DrawerHeader';
+import {Drawer} from './lib/Drawer';
+import {DrawerHeader} from './lib/DrawerHeader';
 export {Drawer, DrawerHeader};
```
Comment (RayRedGoose) [inline modules/labs-react/drawer/index.ts:3]:
```suggestion
```

**Review by RayRedGoose — COMMENTED** [review]
Great work Manny! I just suggest to remove some imports because now it doesn't have sense to be as you switch to named import. Otherwise everything looks good for me.

File: modules/react/loading-animation/index.ts [hunk 12]
```diff
@@ -1,5 +1,4 @@
-import LoadingDots from './lib/LoadingDots';
+import {LoadingDots} from './lib/LoadingDots';
 
 export {LoadingDots as LoadingAnimation, LoadingDots};
-export default LoadingDots;
 export * from './lib/LoadingDots';
```
[the remark this answers is not in the document]
Comment (mannycarrera4) [reply modules/react/loading-animation/index.ts:4]: hmm, I'm also confused, looking at storybook it looks like it uses `LoadingAnimation` I'll update this

File: modules/codemod/lib/v8/spec/removeDefaultImports.spec.ts [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+    import { Select } from '@workday/canvas-kit-react/select';
+    import { SidePanel } from '@workday/canvas-kit-react/side-panel';
+    import { StatusIndicator } from '@workday/canvas-kit-react/status-indicator';
+    import { Switch } from '@workday/canvas-kit-react/switch';
+    import { Table } from '@workday/canvas-kit-react/table';
+    import { TextInput } from '@workday/canvas-kit-react/text-input';
+    `;
+
+    expectTransform(input, expected);
+  });
```
Comment (NicholasBoll) [inline modules/codemod/lib/v8/spec/removeDefaultImports.spec.ts:52]: I think you should add renamed default imports to all these. Something like:

```tsx
  it('should replace renamed default exports to renamed named exports', () => {
    const input = stripIndent`
      import CanvasToast from '@workday/canvas-kit-react/toast';
    `;

    const expected = stripIndent`
      import {Toast as CanvasToast} from '@workday/canvas-kit-react/toast';
    `;
  });
```

**Review by NicholasBoll — APPROVED** [review]

**Pull request merged**